authorization-service/package.json

```diff
@@ -0,0 +1,38 @@
+{
+    "name": "authorization-service",
+    "version": "1.0.0",
+    "description": "Serverless aws-nodejs-typescript template",
+    "main": "serverless.ts",
+    "scripts": {
+        "test": "echo \"Error: no test specified\" && exit 1",
+        "deploy": "sls deploy",
+        "remove": "sls remove --region eu-west-1",
+        "build": "sls package"
+    },
+    "engines": {
+        "node": ">=14.15.0"
+    },
+    "dependencies": {
+        "@middy/core": "^1.5.2",
+        "@middy/http-json-body-parser": "^1.5.2",
+        "source-map-support": "^0.5.19"
+    },
+    "devDependencies": {
+        "@serverless/typescript": "^2.23.0",
+        "@types/aws-lambda": "^8.10.71",
+        "@types/node": "^14.14.25",
+        "json-schema-to-ts": "^1.5.0",
```

Comment:
revisit some dependencies. json-schema-to-ts - not used in this service

Reply:
Ye, there were some redundant dependencies, that came up with the `aws-nodejs-typescript` template. The same for `middyfy`. Removed, thanks 🙂